docs/how-to/best-practices-for-nft-data.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Best Practices for Storing NFT Data using IPFS
+description: Non-fungible tokens can represent all kinds of digital media. Learn some best practices for storing NFT data using IPFS.
+date: 2021-04-02
+---
+
+# Best Practices for Storing NFT Data using IPFS
+
+IPFS is a great fit for storing and addressing data for NFTs, or non-fungible tokens. This guide is all about how to store the data for NFTs on IPFS so that NFT creators and owners have a great experience that can stand the test of time.
+
+Since an NFT can't be easily changed after it's been created, it's a good idea to think about how the data for your NFTs is stored, addressed, and made persistent over time. That's why we'll be getting into the specifics of [how to prepare your NFT metadata](#metadata), and we'll also look at [the different kinds of links to IPFS content](#types-of-ipfs-links-and-when-to-use-them) and when you should use each one. Finally, we'll see why [making a plan for your data's persistence](#persistence-and-availability) is important for a good user experience. By following these recommendations, you can help ensure a long and healthy future for your NFT data.
+
+This guide is aimed at developers building NFT platforms and other tools, and it's focused on how to format your data and link to it for the best long-term results. If you're looking for details about smart contract interactions and how token minting works, head over to our [guide to minting NFTs with IPFS][docs-mint-nfts], where we go over the whole process from end-to-end using an Ethereum test network.
+
+## Types of IPFS links and when to use them
+
+There are a few different ways to refer to data on IPFS, each of which is best suited to different use cases.
+
+### CID
+
+First, there's the "raw" CID, which uniquely identifies a specific piece of content. CIDs can be stored and sent over the network in a compact binary form, but they're represented as strings of random-seeming characters when displayed to users. Here's an example:
```

Comment:
We can cut the first sentence about “raw.” The second sentence is a great initial statement and we don’t need to get into codecs.